import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/CStoreService.java

```diff
@@ -92,13 +92,16 @@ protected void store(
         HttpRequest httpRequest = requestFactory.buildPostRequest(url, content);
         httpRequest.execute();
       } catch (IOException e) {
-        if (e instanceof HttpResponseException &&
-            ((HttpResponseException) e).getStatusCode() ==
-                HttpStatusCodes.STATUS_CODE_SERVICE_UNAVAILABLE) {
-          throw new DicomServiceException(Status.OutOfResources, e);
-        } else {
-          throw e;
+        if (e instanceof HttpResponseException) {
```

Comment:
Can we use DicomWebClient instead here for making the stow request, and make changes so that STOW maps these status codes in the DicomWebClient?

Reply:
Ok